modules/handlers/command_handlers.py

```diff
@@ -208,6 +210,34 @@ def clean_pending_cmd(update: Update, context: CallbackContext):
         clean_pending_job(context=context)
 
 
+def purge_cmd(update: Update, context: CallbackContext):
+    """Handles the /purge command.
+    Deletes all posts and the related votes in the database whose actual telegram message could not be found
+
+    Args:
+        update (Update): update event
+        context (CallbackContext): context passed by the handler
+    """
+    info = EventInfo.from_message(update, context)
+    if info.chat_id == config_map['meme']['group_id']:  # you have to be in the admin group
+        published_memes = DbManager.select_from("published_meme")
+        total_posts = len(published_memes)
+        lost_posts = 0
+        for published_meme in published_memes:
+            try:
+                message = info.bot.forward_message(info.chat_id,
+                                                   from_chat_id=published_meme['channel_id'],
+                                                   message_id=published_meme['c_message_id'])
```

Comment:
maybe here we can put `disable_notification=true` to avoid the spam on our telegram notifications?